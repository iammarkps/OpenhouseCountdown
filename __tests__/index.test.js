import React from 'react'
import { shallow } from 'enzyme'
import renderer from 'react-test-renderer'

import Index from '../pages/index'

describe('Shallow render', () => {
  it('App shows text', () => {
    const app = shallow(<Index />).dive()

    expect(app.find('p').text()).toEqual('Triam Udom Open House 2019')
  })
})

describe('Snapshot', () => {
  it('App match snapshot', () => {
    const component = renderer.create(<Index />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
