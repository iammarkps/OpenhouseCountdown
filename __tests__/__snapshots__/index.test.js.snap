// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot App render 1`] = `
<div
  className="jsx-3937719320"
>
  <div
    className="jsx-3937719320 container"
  >
    <div
      className="jsx-3937719320 introduction"
    >
      <span>
        98
        :
        00
        :
        15
        :
        34
      </span>
    </div>
    <div
      className="jsx-3937719320 footer"
    >
      <div
        className="jsx-3937719320 left"
      >
        <p
          className="jsx-3937719320"
        >
          Triam Udom Open House 2019
        </p>
      </div>
      <div
        className="jsx-3937719320 right"
      >
        <a
          className="jsx-3937719320"
          href="http://www.triamudom.ac.th/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Triam Udom
        </a>
        <a
          className="jsx-3937719320"
          href="https://www.facebook.com/TriamUdomOPH/"
          rel="noopener noreferrer"
          target="_blank"
        >
          Facebook
        </a>
        <a
          className="jsx-3937719320"
          href="https://www.facebook.com/triamudomclubs/"
          rel="noopener noreferrer"
          target="_blank"
        >
          TUCMC
        </a>
      </div>
    </div>
  </div>
</div>
`;
